packages/storybook/context/logger.context.js

```diff
@@ -8,17 +8,17 @@ export default () => {
     };
 
     const logger = {
-        logError: (logMessage, store, logPayload = {}) => {
+        error: (logMessage, exception, tags, logPayload = {}) => {
             appendStandardProperties(logPayload, logMessage, 'Error', 'client');
```

Comment:
is it ok that this is different to the one above?

Above: `error: (logMessage, error, tags, logPayload = {}) => {`
Here: `error: (logMessage, exception, tags, logPayload = {}) => {`

Reply:
Good shout, I changed everything to exception, thanks!